### DISCUSSION TITLE
Diehl Hydrus Meter Support

### DISCUSSION THREAD

**Discussion by KeenanGeorge03** [wmbusmeters/wmbusmeters#624] [General]
Hi 

Is it possible to add driver for the below meter.

4E4424230694766524067AEB0040256FD25490DADF6DA5B57925204EE69B030A3B447D6FC841BFB993373D1D4DF362CD1D35B74E82C9CFA8AFC4B4CEB37B67A8FEF3CCBBA66D6C08A8BCA7A6C9A60B

**Comment by americanium** [comment]
Hi,
afaik the Diehl Hydrus is supported.
I also have this type and it works fine.
BR,
Alex

**Reply by KeenanGeorge03** [reply]
Hi on my side it say driver unknown

**Reply by johann1956** [reply]
@americanium Hi Alex!
My Post #647 with Problem "connection refused..." how did you configure your Device? 
Thanks for your help!
Johann

**Comment by americanium** [comment]
But you find the meter when doing wmbusmeters auto:t1 ?

**Reply by KeenanGeorge03** [reply]
Hi apologies for the delayed response this in auto below please find two telegrams one working and another not working same meter type but different meters

**Working**

404424232407016441487AE6903258DCFEBC2F4A672E176E30AEFDB897EBDB8A3588BEED552168C6DBEB3857B4F0B93EE82346D50E9295883CCDCCFEF9D9559EE8

**Not Working**

4E4424230894766524067A6E004005901EA1F202ACCABD5E9DCF63D5C6700875FC56E02A6FC7B62CBEA7DC3816F39877729A959BAE320F5CD2F4A67DA4B35FD464094A3445F84CC7780201301CCD5D
<img width="704" alt="Analyze" src="https://user-images.githubusercontent.com/15806079/197953654-f5e150a3-5486-4eac-80b4-851b80b0e66c.PNG">

**Comment by KeenanGeorge03** [comment]
Hi @americanium 

Please see below if i use auto:t1
<img width="664" alt="t1" src="https://user-images.githubusercontent.com/15806079/198076289-bda33abb-8ca7-4eb9-a7a1-ad67a3e05269.PNG">

**Reply by KeenanGeorge03** [reply]
Hi @americanium hope you well.

Do you perhaps have any feedback on the above

Regards

**Reply by KeenanGeorge03** [reply]
Hi Guys

Hope you are well.

Any update on the above.

Regards

**Comment by americanium** [comment]
Hi!
Sorry, so much to do at the moment....

I don't know why you have to add the 4E..... number ... my test-pi is currently offline so i can't check my exact parameters. 
I'll do asap for you - but i would first try it without the 4E... number ...

BR
